io.qt.QtDesignStudio.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: io.qt.QtDesignStudio
+runtime: org.kde.Sdk
+runtime-version: '6.5'
```

Comment:
this runtime version is EOL. So either this has to use a more recent runtime, or this submission has to wait to be fixed.

Reply:
Hmm, 6.7 is broken, what about 6.6?

Reply:
How is it broken?

Reply:
Behavior changes. Qt is not behavior-compatible. They are small but lead to a subpar experience. Actually, very complex software will suffer from it, because it is really hard to change something without breaking something: https://www.hyrumslaw.com/

Reply:
6.6 is going to be EOL ed 2 months.